emokabe/CapstoneProject
# Current course posts

### Context
This functionality adds on to #14, which implemented post-to-course_id mappings. The user can now include a title on their post, and the app can now display the posts for a specific course that corresponds to the course_id stored locally. The question title, question body, and course_id are stored in the “message” field of the Facebook Group post, in the format “QuestionTitle/0\n\nQuestionBody/0\n\nCourseTag1,CourseTag2” with ‘/0’ as the delimiter and the newlines to make the posts on the Facebook Group itself more presentable.

I have yet to add the functionality for selecting between courses, but will PR this part first to avoid PRing too much code at once.

### Test plan
1. Run the simulator, then click the login button to login with Facebook credentials.
2. After seeing the Home Screen with posts, press on the right navigation bar button that reads “Post”
3. Write the question title in the “Title” field and question body in the “Write something… “ field, then tap “Post”.
4. You will see the timeline pop up, with the new post at the top.
5. Go to the Facebook Group that stores these posts, and see that there is a new post formatted in the format mentioned in the Context.

### MVP features implemented
This PR is part of the “Select a course from a list to view its feed” feature.

https://user-images.githubusercontent.com/107252243/180103574-6c9e3b13-722d-4fe0-9c9e-91cd4981d78a.mov
